src/services/study/network-map.js

```diff
@@ -92,7 +92,7 @@ export function fetchEquipmentsIds(
     nominalVoltages = null
```

Comment:
```suggestion
    nominalVoltages = undefined
```
Do not modify `getQueryParamsList` which is used in a lot of places